src/js/tracks/text-track.js

```diff
@@ -336,7 +336,17 @@ class TextTrack extends Track {
    * @param {TextTrack~Cue} cue
    *        The cue to add to our internal list
    */
-  addCue(cue) {
+  addCue(_cue) {
+    let cue = _cue;
+
+    if (!(_cue instanceof window.vttjs.VTTCue)) {
+      cue = new window.vttjs.VTTCue(_cue.startTime, _cue.endTime, _cue.text);
```

Comment:
@misteroneill sort of. Given that we're using a custom fork, we can do what we want with it at this point.

The only thing I can imagine that would be tricky to get right to have vttjs not export itself globally is getting the novtt build working correctly.